src/Logging.AzureAppServices/src/BatchingLogger.cs

```diff
@@ -28,7 +28,7 @@ public bool IsEnabled(LogLevel logLevel)
         return _provider.IsEnabled;
     }
 
-    public void Log<TState>(DateTimeOffset timestamp, LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+    public void Log<TState>(DateTimeOffset timestamp, LogLevel logLevel, EventId _, TState state, Exception exception, Func<TState, Exception, string> formatter)
```

Comment:
Shouldn't the EventId get logged?

Reply:
This method turns the log to a string. We could add it to the log, but I'm not super familiar who the consumer is and if changing the format would affect it in any way.